network-api/networkapi/buyersguide/utils.py

```diff
@@ -12,3 +15,29 @@ def quad_to_tri(input):
     if input == 'No':
         return False
     return None
+
+
+class AlterModelBases(ModelOperation):
```

Comment:
I added this as a util function rather than putting this in the migration itself, *just in case* we need it in the future, at which point we can move it into the utils file for "everything" rather than just "the buyers guide"